code/extensions/ContentReviewCMSExtension.php

```diff
@@ -67,7 +67,7 @@ public function save_review($data, Form $form) {
 		if(!isset($data['ID'])) {
 			throw new SS_HTTPResponse_Exception("No record ID", 404);
 		}
-		$SQL_id = Convert::raw2sql($data['ID']);
+		$SQL_id = (int) $data['ID'];
```

Comment:
Nice. :)

I should add a phpdoc to the DataList methods `don't raw2sql data here`